src/connectors/onedrive/connector.py

```
@@ -261,13 +275,15 @@ async def _async_sync():
                         doc = await self.get_file_content(file_id)
                         self.emit(doc)
                     except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)